sig/vendor/multipart-post/sig.rbs

```diff
@@ -0,0 +1,16 @@
+module Net
```

Comment:
multipart-post library and community definition don't have RBS for it. This is required to pass check by Steep.